services/workflows-service/src/business-report/dto/create-business-report.dto.ts

```diff
@@ -0,0 +1,56 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsIn, IsOptional, IsString, MinLength } from 'class-validator';
+import { countryCodes } from '@ballerine/common';
+import { BusinessReportType } from '@prisma/client';
+import type { ObjectValues } from '@/types';
+
+export class CreateBusinessReportDto {
```

Comment:
Don't think class-validator can express company name or business id like zod, this is handled in the endpoint itself.